### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by LauraBeatris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
-Documentaçäo em desenvolvimento 
+<p align="center">
+<img src=".github/logo.png" width="400" alt="Front-end Brasil">
+</p>
+<h1 align="center">Fórum</h1>
+<p align="center">🦖 Discussões em formato de issues impulsionadas pela comunidade brasileira de Deno</p>
+
+> Disclaimer: Não nos resposabilizamos por comentários de má conduta, os contribuidores possuem responsabilidade pelo seus atos e estes não possuem vinculo com a organização.
```
Comment (pedro-lb) [inline README.md:7]: Typo

```suggestion
> Disclaimer: Não nos responsabilizamos por comentários de má conduta, os contribuidores possuem responsabilidade pelo seus atos e estes não possuem vínculo com a organização.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
-Documentaçäo em desenvolvimento 
+<p align="center">
+<img src=".github/logo.png" width="400" alt="Front-end Brasil">
```
Comment (pedro-lb) [inline README.md:2]: 🍻 

```suggestion
<img src=".github/logo.png" width="400" alt="Deno Brasil">
```

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+
+Um dos nossos maiores objetivos é incentizar a comunidade brasileira de Deno a impulsionar projetos open source, e para isso, estamos desenvolvemos outros projetos:
+
+- [Contribua para projetos open source](https://github.com/DenoBrazil/open-source): Lista de bibliotecas e projetos open-source que estão contribuindo efetivamente para o ecossistema. Você pode ficar de olho na issues de um desses repositórios e contribuir com discussões ou até mesmo, enviar pull requests 🚀.
+
+- [Site Oficial do Deno Brazil](https://github.com/DenoBrazil/denobrazil.com): Repositório do nosso site oficial onde iremos colocar informações de eventos e noticias sobre a comunidade.
+
+## 🎙 Participe da comunidade ativamente
+
+- [Entre na nossa comunidade do Discord](https://discord.gg/FAJ8Bc)
```
Comment (pedro-lb) [inline README.md:27]: This link does not expire right?

File: README.md (hunk 2 above)
Comment (LauraBeatris) [reply README.md:2]: 👀

File: README.md (hunk 3 above)
Comment (LauraBeatris) [reply README.md:27]: Good question 🤔

I found that:

> There are two situations in which your Discord invites may appear as expired:
> 
> 1. You were IP banned from the server.
> 2. There was a time limit on the invite link that was created. Default is 24 hours if I’m not mistaken.

Refer to: https://www.quora.com/Why-do-all-of-my-Discord-invites-appear-expired

Comment (LauraBeatris) [reply README.md:27]: Look!! 
We are able to choose when the link is going to expire and also the limit of usage 

Screen Shot 2020-05-25 at 11 06 50

File: README.md (hunk 1 above)
Comment (LauraBeatris) [reply README.md:7]: Grammar is not my thing 🙃

File: README.md (hunk 2 above)
Comment (LauraBeatris) [reply README.md]: Fixed in b39bed2893b786ffef2419e722ab7a9c6961f30f

File: README.md (hunk 3 above)
Comment (LauraBeatris) [reply README.md]: Fixed in b39bed2893b786ffef2419e722ab7a9c6961f30f

File: README.md (hunk 1 above)
Comment (LauraBeatris) [reply README.md]: Fixed in 88252b67fe59bba7aa32d2bd678ce0703b3d2fad

**Comment by pedro-lb** [comment]
We should add more badges to the start of our readme

Use https://shields.io/

- Discord link
- Website link
- Forum link
- Github organization link
- Anything else

**Pull request merged**